Core/Base/ElasticDB.py

```diff
@@ -0,0 +1,96 @@
+########################################################################
+# $Id: $
+########################################################################
+
+""" ElasticDB is a base class used to connect an Elasticsearch database and manages 
+queries.
+"""
+
+__RCSID__ = "$Id$"
+
+from DIRAC                                       import gLogger, gConfig
+from DIRAC.Core.Utilities.ElasticSearchDB        import ElasticSearchDB
+from DIRAC.ConfigurationSystem.Client.Utilities  import getElasticDBParameters
+
+class ElasticDB( ElasticSearchDB ):
```

Comment:
I fail at seeing the interest of this class alltogether. Why not using directly ```ElasticSearchDB``` ?